Clarify how to add HTTP headers in the dashboard (#18558)
cloudflare/cloudflare-docs: src/content/docs/logs/get-started/enable-destinations/http.mdx

5. In **Select a destination**, choose **HTTP destination**.

6. Enter the **HTTP endpoint** where you want to send the logs to, and select **Continue**.

> 		* You can use `"header_*"` URL parameters to set request headers (for example, to pass an authentication token to your HTTP endpoint).

Reviewer: ```suggestion
		- You can use `"header_*"` URL parameters to set request headers, such as passing an authentication token to your HTTP endpoint.
```
